src/test/java/com/mentalhealthapi/mentalhealthapi/MentalHealthApiApplicationTests.java

```diff
@@ -41,6 +41,12 @@ private void whenSearchDisorderWithNameDepression() {
     }
 
     private void ReturnsDepressionForNameDepression() {
```

Comment:
While you're here, remember that method names should be lowerCamelCase.  Thus, this should be returnsDepressionForNameDepression.

Reply:
I fixed this some time ago. Thank you